apps/crc_usage.py

```diff
@@ -37,4 +36,5 @@ def app_logic(self, args: Namespace) -> None:
         if not account_exists:
             raise RuntimeError(f"No slurm account was found with the name '{args.account}'.")
 
-        print(Shell.run_command(f'{self.banking_executable} {args.account}'))
+        account = AccountServices(args.account)
+        print(account._build_usage_table())
```

Comment:
We may want to revisit the usage table output or make a different function in the bank for constructing what's printed here (end users see this) if we want it to be different from what admins see when using crc-bank directly from the command line.